eventstore/tests/test_views.py

```diff
@@ -1283,38 +1283,43 @@ def test_successful_inbound_messages_request(self, mock_handle_inbound):
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         [messages] = Message.objects.all()
         self.assertEqual(str(messages.contact_id), "sender-wa-id")
-        self.assertEqual(
-            messages.timestamp, datetime.datetime(2018, 2, 15, 11, 38, 20, tzinfo=UTC)
-        ),
-        self.assertEqual(messages.id, "9e12d04c-af25-40b6-aa4f-57c72e8e3f91"),
-        self.assertEqual(messages.type, "image"),
-        self.assertEqual(messages.message_direction, Message.INBOUND),
-        self.assertEqual(
-            messages.data,
-            {
-                "context": {
-                    "from": "sender-wa-id-of-context-message",
-                    "group_id": "group-id-of-context-message",
-                    "id": "message-id-of-context-message",
-                    "mentions": ["wa-id1", "wa-id2"],
-                },
-                "image": {
-                    "file": "absolute-filepath-on-coreapp",
-                    "id": "media-id",
-                    "link": "link-to-image-file",
-                    "mime_type": "media-mime-type",
-                    "sha256": "checksum",
-                    "caption": "image-caption",
-                },
-                "location": {
-                    "address": "1 Hacker Way, Menlo Park, CA, 94025",
-                    "name": "location-name",
+        (
+            self.assertEqual(
+                messages.timestamp,
+                datetime.datetime(2018, 2, 15, 11, 38, 20, tzinfo=UTC),
+            ),
+        )
+        (self.assertEqual(messages.id, "9e12d04c-af25-40b6-aa4f-57c72e8e3f91"),)
+        (self.assertEqual(messages.type, "image"),)
+        (self.assertEqual(messages.message_direction, Message.INBOUND),)
```

Comment:
This looks like `ruff` has helpfully turned a bunch of accidental trailing commas into explicit tuples. Perhaps we should remove the commas instead?

Reply:
Fixed

Reply:
I think there are more of these -- mostly multiline ones. Any line that starts `(self.assert...` or the multiline equivalent with the `(` on the line above can be de-tupled.